### ISSUE TITLE
Smoldering Egg of Millagazor not tracking

### ISSUE THREAD

**Issue by Headpiker** [WowRarity/Rarity#341]
If you are doing the Firelands skip on Alysrazor fight with the feathers and fly straight to Ragnaros, Rarity isn't tracking tries for Smoldering Egg of Millagazor. This is probably since you are already in combat with Alysrazor. Is there a way to fix this, so that Rarity keeps track on Smoldering Egg aswell?

**Comment by Duckwhale** [comment]
Are you certain this is only happening when still being in combat? As far as I know, tracking for Firelands is generally broken due to a Blizzard bug.

Since that item uses kill statistics, the attempt should always be added after you drop combat. If it wasn't for the bug, that is; I'd expect any attempt to not count currently because the kill statistics aren't updated properly (by Blizzard).

**Comment by Headpiker** [comment]
Yes, certain. I have tracked kills of the boss and I can confirm it only happens when im flying over with feathers.

**Comment by Duckwhale** [comment]
Seems odd, but without additional information (debug log?) I don't know what's wrong and I don't currently have time to investigate myself :(
